roles/php/tasks/install-RedHat.yml

```diff
@@ -1,19 +1,39 @@
 ---
 
+- name: Ensure Repositories are installed (RedHat)
+  yum:
+    name: "{{ item }}"
+    state: present
+  loop: "{{ php_yum_repositories }}"
+  when: php_alt_repo and php_yum_repositories is defined and php_yum_repositories|length
+  become: true
+
+- name: Import remi GPG key.
+  rpm_key:
+    key: "{{ item }}"
+    state: present
+  loop: "{{ php_yum_keys }}"
+  when: php_alt_repo and php_yum_repositories is defined and php_yum_repositories|length
+  become: true
+
 - name: Ensure PHP packages are installed (RedHat)
   yum:
-    name: "{{ php_packages }}"
+    enablerepo: "remi,remi-safe,remi-php{{ php_version }}"
+    name: "{{ item }}"
     state: present
     update_cache: yes
   become: true
+  loop: "{{ php_default_packages }}"
```

Comment:
this should `php_packages`. see task in `tasks/main.yml`:
```
- name: Define PHP packages
  set_fact:
    php_packages: "{{ php_default_packages | list }}"
  when: php_packages is not defined
```

Reply:
Done